.eslintrc.json

```diff
@@ -1,9 +1,5 @@
 {
   "parser": "babel-eslint",
-  "env": {
```

Comment:
This is why the Codacy checks failed. We must let the linter know that we are using es6 and browser-specific features.